lib/data_grid.dart

```diff
@@ -629,3 +642,40 @@ enum SortingState {
     }
   }
 }
+
+class SortingIconSettings {
+  final IconData ascending;
+  final IconData descending;
+  final double size;
+  final Color? color;
+  final EdgeInsets padding;
+
+  const SortingIconSettings({
+    this.ascending = Icons.arrow_upward_rounded,
+    this.descending = Icons.arrow_downward_rounded,
+    this.size = 20,
+    this.color,
+    this.padding = const EdgeInsets.symmetric(horizontal: 4.0),
+  });
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other)) return true;
+
+    return other is SortingIconSettings &&
+        other.ascending == ascending &&
+        other.descending == descending &&
+        other.size == size &&
+        other.color == color &&
+        other.padding == padding;
+  }
+
+  @override
+  int get hashCode {
+    return ascending.hashCode ^
+        descending.hashCode ^
+        size.hashCode ^
+        color.hashCode ^
+        padding.hashCode;
```

Comment:
```suggestion
    return Object.hash(ascending, descending, size, color, padding);
```
I think this is better, the package freezed uses it internally. Syntax might be wrong as I just did this in github